src/index.html

```diff
@@ -22,6 +22,388 @@
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </section>
+
+          <p class="card__review-count">Reviews: 5</p>
+        </section>
+
+        <section class="card__price-container">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price">$2,199</p>
+        </section>
+
+        <a
+          class="card__buy"
+          href="#"
+        >
+          BUY
+        </a>
+      </section>
+      <section class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="iMac A1419"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <section class="card__review-container">
+          <section class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </section>
+
+          <p class="card__review-count">Reviews: 5</p>
+        </section>
+
+        <section class="card__price-container">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price">$2,199</p>
+        </section>
+
+        <a
+          class="card__buy"
+          href="#"
+        >
+          BUY
+        </a>
+      </section>
+      <section class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="iMac A1419"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <section class="card__review-container">
+          <section class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </section>
+
+          <p class="card__review-count">Reviews: 5</p>
+        </section>
+
+        <section class="card__price-container">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price">$2,199</p>
+        </section>
+
+        <a
+          class="card__buy"
+          href="#"
+        >
+          BUY
+        </a>
+      </section>
+      <section class="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="iMac A1419"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <section class="card__review-container">
```

Comment:
the section is usually something bigger (that has its own title). It's not a mistake here and you can keep the section, just remember that div is also allowed to use 🙂